BonusCalcListener.Tests/Gateway/OperativesGatewayTests.cs

```diff
@@ -0,0 +1,74 @@
+using BonusCalcListener.Gateway;
+using BonusCalcListener.Gateway.Interfaces;
+using BonusCalcListener.Infrastructure;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NuGet.Frameworks;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BonusCalcListener.Tests.Gateway
+{
+    public class OperativesGatewayTests : DatabaseTests
+    {
+        private Mock<ILogger<OperativesGateway>> _loggerMock;
+        private OperativesGateway _classUnderTest;
+
+        [SetUp]
+        public void Setup()
+        {
+            _loggerMock = new Mock<ILogger<OperativesGateway>>();
+            _classUnderTest = new OperativesGateway(BonusCalcContext, _loggerMock.Object);
+        }
+
+        [Test]
+        public async Task RetrievesOperativeIfOperativeFound()
```

Comment:
I would suggest changing this title to ActivatesOperativeIfArchivedOperativeFound

Reply:
renamed